<ui version="4.0" >
 <class>CWSelectDirsPage</class>
 <widget class="QWidget" name="CWSelectDirsPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTreeView" name="treeView" />
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
